modules/disk-uploader/pkg/image/image.go

```diff
@@ -60,15 +61,20 @@ func Build(diskPath string, config v1.Config) (v1.Image, error) {
 	return image, nil
 }
 
-func Push(image v1.Image, imageDestination string, pushTimeout int) error {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*time.Duration(pushTimeout))
+func Push(image v1.Image, imageDestination string, pushTimeout string) error {
+	timeout, err := strconv.Atoi(pushTimeout)
```

Comment:
nit: I would add this to Init function in modules/disk-uploader/pkg/parse/clioptions.go file

Reply:
+1